sdcm/utils/database_query_utils.py

```diff
@@ -181,6 +181,45 @@ def get_table_clustering_order(ks_cf: str, ck_name: str, session) -> str:
     return clustering_order
 
 
+def is_system_keyspace(keyspace: str) -> bool:
+    system_keyspace_prefixes = ("system", "alternator_usertable", "audit")
+    return keyspace.startswith(system_keyspace_prefixes)
+
+
+def get_max_replication_factor(session) -> (int, str):
+    """
+     returns the user keyspace with the highest replication-factor value.
+    """
+    query = "SELECT keyspace_name, replication FROM system_schema.keyspaces"
+    cql_result = session.execute(query)
+
+    max_replication_factor = 0
+    keyspace_with_max_rf = None
```

Comment:
`keyspace_with_max_rf` as `None` is not expected in above code logic.
Probably set it to some string?

Reply:
fixed